google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2002,6 +2004,23 @@ def iter(self):
                 # API client doesn't support this resource, ignore.
                 LOGGER.debug(e)
 
+class GsuiteGroupsSettingsIterator(ResourceIterator):
+    """The Resource iterator implementation for Gsuite Group Settings"""
+
+    def iter(self):
+        """Resource iterator.
+
+        Yields:
+            Resource: GsuiteGroupsSettings created
+        """
+        gsuite = self.client
+        try:
+            data = gsuite.fetch_gsuite_groups_settings(self.resource['email'])
```

Comment:
@blueandgold please see line 1957 (it wont let me comment on unchanged lines) and let me know what the purpose of `if self.resource.has_directory_resource_id()` is in Gsuite Group Iterator?  I want to double check I dont need something similar here